hammers/scripts/clean_old_aggregates.py

```diff
@@ -111,7 +111,7 @@ def has_active_allocation(orph):
         alloc for alloc in host_allocs
         if alloc['resource_id'] == orph
     ]
-    if not matching_allocs:
```

Comment:
Can you reason as to why this is failing now?

Reply:
I am more confused about how it worked before. My understanding is matching_allocs should always return something for a node, and if there are no allocations, it is just an empty list. There may have been a change to blazar where allocations before only included hosts with reservations, but I am uncertain of when this would have occurred.